android/src/main/kotlin/com/bitmovin/player/flutter/FlutterPlayerView.kt

```diff
@@ -77,6 +92,14 @@ class FlutterPlayerView(
         }
 
         activityLifecycle.addObserver(activityLifecycleObserver)
+        activity.registerComponentCallbacks(configurationChangedCallback)
+    }
+
+    private fun onPictureInPictureModeChanged(
+        isInPictureInPictureMode: Boolean,
+        newConfig: Configuration,
+    ) {
+        // TODO: Handle picture in picture mode changed
```

Comment:
I guess this `TODO` will be part of a future PR, right?

Reply:
Yes 🙂